website/docs/r/kusto_cluster.html.markdown

```diff
@@ -46,6 +46,10 @@ The following arguments are supported:
 
 * `sku` - (Required) A `sku` block as defined below.
 
+* `allowed_fqdn_list` - (Optional) List of allowed FQDNs(Fully Qualified Domain Name) for egress from Cluster.
+
+* `allowed_ip_rangelist` - (Optional) The list of ips in the format of CIDR allowed to connect to the cluster.
```

Comment:
should be `allowed_ip_rangelist_list` (you are missing underscore between range and list)

Reply:
oh, let me change that typo